home/templates/home/new.html

```diff
@@ -21,15 +21,14 @@
             <img class="media-object" data-author-id="{{post.authorid}}"  src="{{ post.avatar }}" width="64px" height="64px">
           </a>
           <div class="media-body">
-            <h4 class="media-heading listing-title"><a href="/post/{{ post.slug }}/view">{{ post.title }}</a></h4><div><a href="/post/{{ post.slug }}" class="listing-comments">
-            {% if post.comments|length == 1 %}
-                View 1 comment
-            {% elif post.comments %}
-                View {{ post.comments|length }} comments
-            {% else %}
-                Join the discussion
-            {% endif %}
-              </a></div>
+            <h4 class="media-heading listing-title">
+              <a href="/post/{{ post.slug }}/view">{{ post.title }}</a>
+            </h4>
+            <div>
+              <a href="{{ post.title|zulip_url:post.stream }}" target="_blank" class="listing-comments">
+                Join the discussion (on Zulip)
```

Comment:
I'm not too attached to the old wording "Join the discussion". Happy to simplify this down to "Comments on Zulip" or "Discuss on Zulip"